tests/components/laundrify/conftest.py

```diff
@@ -0,0 +1,57 @@
+"""Configure py.test."""
+import json
+from unittest.mock import patch
+
+import pytest
+
+from .const import VALID_ACCESS_TOKEN, VALID_ACCOUNT_ID
+
+from tests.common import load_fixture
+
+
+@pytest.fixture(name="laundrify_setup_entry")
+def laundrify_setup_entry_fixture():
+    """Mock laundrify setup entry function."""
+    with patch(
+        "homeassistant.components.laundrify.async_setup_entry", return_value=True
+    ) as mock_setup_entry:
+        yield mock_setup_entry
+
+
+@pytest.fixture(name="laundrify_api_mock", autouse=True)
+def climacell_config_entry_update_fixture():
```

Comment:
whoops, looks like an errant copy 😉

Reply:
In addition, you can use fixtures in other fixtures. So add `laundrify_exchange_code` as a parameter here and you should be able to get rid of one of the patch statements. Same with `laundrify_validate_token`

Reply:
Nice! I start to like this language 😄

Fixed in 92283ac621970c560a1f186f310dbb2385236881